sdk/state/ingest.go

```diff
@@ -28,7 +31,13 @@ func (c *Channel) IngestTx(txXDR, resultXDR, resultMetaXDR string) error {
 		return fmt.Errorf("transaction unrecognized")
 	}
 
-	c.ingestTxToUpdateInitiatorEscrowAccountSequence(tx)
+	err = c.ingestTxToUpdateInitiatorEscrowAccountSequence(tx)
+	if err != nil {
+		if errors.Is(err, errOldSeqNum) {
+			return nil
+		}
+		return err
+	}
```

Comment:
💡 FYI you can check for the error ahead of other errors, which avoid nesting and keeping the logic a bit flatter.
```suggestion
	err = c.ingestTxToUpdateInitiatorEscrowAccountSequence(tx)
	if errors.Is(err, errOldSeqNum) {
		return nil
	}
	if err != nil {
		return err
	}
```